File: 151051/Makovetskiy/task310+320/dc-rest/src/main/java/com/rmakovetskij/dc_rest/model/Label.java
package com.rmakovetskij.dc_rest.model;

import jakarta.validation.constraints.Size;
import lombok.Data;

import jakarta.persistence.*;   //поменял с javax на jakarta

@Data
@Entity
@Table(name = "tbl_label")
public class Label {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Size(min = 2, max = 32)
    @Column(nullable = false, unique = true)
    private String name;
}

File: 251003/Pankratiev/rest/publisher/src/main/kotlin/bsuir/dc/publisher/entity/Label.kt
package bsuir.dc.publisher.entity

import jakarta.persistence.*

@Entity
@Table(name = "tbl_label", schema = "public")
data class Label(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Long = 0,

    @Column(nullable = false, unique = true, length = 32)
    val name: String  = "",
)
